test/gateway/core.js

```diff
@@ -37,30 +34,72 @@ contract('Core', function (accounts) {
             blockHeight = new BN(5);
             contractsData = await coreUtils.deployCore(artifacts, accounts);
             core = contractsData.core;
-            workersContract = contractsData.workersContract;
             worker = contractsData.worker;
             registrar = contractsData.registrar;
             chainIdRemote = contractsData.chainIdRemote;
-            chainIdOrigin = contractsData.chainIdOrigin;
         });
 
         it('has coreChainIdRemote', async () => {
             assert.equal(await core.chainIdRemote.call(), chainIdRemote);
         });
+    });
 
-        it('has coreChainIdOrigin', async () => {
-            assert.equal(await core.coreChainIdOrigin.call(), chainIdOrigin);
+    describe('setCoCoreAddress', async () => {
+        beforeEach(async () => {
+            contractsData = await coreUtils.deployCore(artifacts, accounts);
+            core = contractsData.core;
+            owner = contractsData.owner;
+            worker = contractsData.worker;
+
+            coCoreAddress = accounts[8];
         });
 
-        it('has workers', async () => {
-            assert.equal(await core.workers.call(), workersContract.address);
-            let latestStateRootBlockHeight = await core.getLatestStateRootBlockHeight.call();
+        it('should allow the organization to set the address', async () => {
+            await core.setCoCoreAddress(coCoreAddress, { from: owner });
```

Comment:
There is not assert for this test if it is successful and actually set the coCore address.

Reply:
added